packages/d3fc-webgl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@d3fc/d3fc-webgl",
+  "version": "0.0.1",
+  "description": "A collection of utilities to enable drawing to WebGL canvases",
+  "license": "MIT",
+  "keywords": [
+    "d3",
+    "d3fc",
+    "webgl"
+  ],
+  "main": "build/d3fc-webgl.js",
+  "module": "index",
+  "scripts": {
+    "bundle": "d3fc-scripts bundle",
```

Comment:
Flagging this now - will need to change once master is webgl branch is rebased off master